### DISCUSSION TITLE
Code review IM-12

### DISCUSSION THREAD

**Discussion by YanPetrov7** [HowProgrammingWorks/LiveQA#52] [Code review]
Перший курс, початок курсової роботи.
Пишу гру "Bunker". Грають від 8 до 16 гравців. Мета - потрапити в бункер з характеристиками які згенерувалися в програмі. Гра поділена на раунди, в яких гравці відкривають по одній характеристиці та проводять голосування, в якому одного з гравців лишають за бункером. В бункері є місце лише для половини гравців, кожному з присутніх потрібно довести свою корисність для подальшого виживання. Гра вважається виграною якщо в бункер потрапили 25% чоловіків та 25% жінок, в бункері нема людей які страждають смертельно небезпечними хворобами та психічними розладами.
Буду вдячний за ревью [кода](https://github.com/YanPetrov7/CardGameBunker/blob/main/main.js).

**Comment by tshemsedinov** [comment]
- Code style
- replaceAll
- Try to optimize long args list

**Reply by YanPetrov7** [reply]
made some [corrections](https://github.com/YanPetrov7/CardGameBunker)

**Comment by tshemsedinov** [comment]
- Use map - https://github.com/YanPetrov7/CardGameBunker/blob/c2e417d5e14d9e480711cedf718a37f79037f2e1/main.js#L98
